entrypoint.sh

```diff
@@ -1,40 +1,57 @@
 #!/bin/bash -e
 
 AWS_REGION="${AWS_REGION:-us-east-1}"
-MISBEHAVING_NOTICE="may be misbehaving. In a perfect world, our monitoring detected this problem and Platform Engineering was alerted... but just in case, please let us know."
+if [ "${APP_ENV}" != "" ]; then
+  export SERVICE_ENV="${APP_ENV}"
+fi
+if [ "${APP_NAME}" != "" ]; then
+  export SERVICE_NAME="${APP_NAME}"
+fi
+if [ "${APP_PRODUCT}" != "" ]; then
+  export SERVICE_PRODUCT="${APP_PRODUCT}"
+fi
```

Comment:
I feel like there has to be a better way do this besides a whole bunch of `if` statements.